components/generic/ImageWithAttribution.stories.js

```diff
@@ -1,14 +1,8 @@
 import { storiesOf } from '@storybook/vue';
 import VueRouter from 'vue-router';
+import VueI18n from 'vue-i18n';
 import ImageWithAttribution from './ImageWithAttribution.vue';
 
-const i18n = {
-  locale: 'en',
-  messages: {
-    en: {}
-  }
-};
-
```

Comment:
instead of getting rid of this just do:
```
const i18n = new VueI18n();
```
So it doesn't have to be initialized twice below